sdk/maps/maps-geolocation-rest/samples-dev/getCountryCodeFromIP.ts

```diff
@@ -1,24 +1,25 @@
 // Copyright (c) Microsoft Corporation.
-// Licensed under the MIT License.
+// Licensed under the MIT license.
+
 import MapsGeolocation, { isUnexpected } from "@azure-rest/maps-geolocation";
-import { AzureKeyCredential } from "@azure/core-auth";
+import { DefaultAzureCredential } from "@azure/identity";
 import * as dotenv from "dotenv";
 
 dotenv.config();
 
 /**
  * @summary This sample demonstrates how to get the country code for an IP address using MapsGeolocation.
  */
-async function successfullyRetrieveCountryCodeFromIPAddress() {
-  /** Use subscription key authentication */
-  const subscriptionKey = process.env.MAPS_SUBSCRIPTION_KEY || "";
-  const credential = new AzureKeyCredential(subscriptionKey);
-  const client = MapsGeolocation(credential);
-
-  /** Or use Azure AD authentication */
-  // const credential = new DefaultAzureCredential();
-  // const mapsClientId = process.env.MAPS_RESOURCE_CLIENT_ID || "";
-  // const client = new MapsGeolocation(credential, mapsClientId);
+async function successfullyRetrieveCountryCodeFromIPAddress(): Promise<void> {
+  /** Use Azure AD authentication (Recommended) */
```

Comment:
Please help to rename "Azure AD" and "AAD" to "Microsoft Entra ID" in all the files.